pkg/network/usm/nodejs.go

```diff
@@ -163,7 +166,7 @@ func (m *nodeJSMonitor) Stop() {
 }
 
 // getNodeJSPath checks if the given PID is a NodeJS process and returns the path to the binary
```

Comment:
The documentation does not fit the function name isNodeJSBinary

Reply:
thanks, fixed